protokube/pkg/protokube/kube_boot.go

```diff
@@ -101,8 +102,23 @@ func (k *KubeBoot) Init(volumesProvider Volumes) {
 
 // RunSyncLoop is responsible for provision the cluster
 func (k *KubeBoot) RunSyncLoop() {
+	ctx := context.Background()
+
+	for {
+		client, err := k.Kubernetes.KubernetesClient()
+		if err != nil {
+			klog.Warningf("could not create kubernetes client: %v", err)
```

Comment:
This one probably needs a sleep and continue?  Or klog.Fatalf?  I think we'll panic on the `client.CoreV1()` call anyway..

Reply:
We may as well panic here. There isn't much we can do if this one fails.